### ISSUE TITLE
Cannot select item when using Talkback screen reader on Android

### ISSUE THREAD

**Issue by nico-amsterdam** [trevoreyre/autocomplete#173]
Tested this on a Android phone width Talkback 14.1 on Chrome .
I can type something in the input field, but if I want to select something from the suggestions it selects the whole box, and it is not possible to select one item.

Possible fix: add tabindex -1 to make the elements focusable
